server.js

```diff
@@ -538,14 +538,21 @@ app.get('/question', function(req, res) {
             return res.status(400).send('Question is not available');
         }
 
-        var answeredList = common.getIdsListFromJSONList(questionFound.correctAttempts);
+        var answeredList = common.getIdsListFromJSONList2(questionFound.correctAttempts);
+        var hasQrating = false;
+        for (var i in questionFound.ratings) {
+            if (questionFound.ratings[i].user === req.session.user.id) {
+                hasQrating = true;
+            }
+        }
         return res.status(200).render('question', {
             user: req.session.user,
             question: questionFound,
             answered: (answeredList.indexOf(req.session.user.id) !== -1),
             isAdmin : function() {
                 return req.session.user.type === common.userTypes.ADMIN;
             },
+            hasQrating: hasQrating/*ADD QUESTION RATING GET FUNCTION HERE*/,
```

Comment:
seems like this was a comment for you, should this be removed/changed?

Reply:
Whoops yeah, after all those merge conflicts on Saturday night this change kept coming up. I shall address this right now.